XIVSlothCombo/Combos/PvE/NIN.cs

```diff
@@ -371,8 +374,8 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                             if (IsEnabled(CustomComboPreset.NIN_ST_AdvancedMode_Ninjitsus_Suiton) &&
                                 setupSuitonWindow && 
                                 TrickAttack.LevelChecked() &&
-                                !HasEffect(Buffs.Suiton) &&
                                 chargeCheck &&
+                                (IsEnabled(CustomComboPreset.NIN_ST_AdvancedMode_Suiton_Uptime) && GetRemainingCharges(Ten) == 2 || (GetRemainingCharges(Ten) == 1 && GetCooldownChargeRemainingTime(Ten) < 3)) &&
```

Comment:
Remove this, you only need the code from L245-247 to enable the uptime feature. This also interferes with people that have the feature enabled to save 1 charge.

Reply:
Without that check, the uptime feature won't function. The Suiton needs to be delayed rather than fired off immediately, as per the description to the new Suiton uptime. It doesn't interfere with ChargeHold, it actually uses the very same check as it.

Reply:
It will work once you add in `IsNotEnabled(CustomComboPreset.NIN_ST_AdvancedMode_Suiton_Uptime` to L191

Reply:
Let me clarify: it does technically work but Suiton normally is fired off immediately once the conditions are met. The feature without this delay is pointless, the delay there is to hold it until we're about to cap. Though thinking about it I should also add one for if trick is about to come off cooldown, which also does ChargeCheck, so that Trick doesn't risk drifting without ChargeHold on...but that's already a monstrous if statement line...